es/lessons/basics/control-structures.md

```diff
@@ -120,7 +120,7 @@ iex> cond do
 "But this will"
 ```
 
-Como `case`, `cond` alzará un error si no hay una coincidencia. Para manejar esto, podemos definir una condición cuyo valor es `true`:
+Como `case`, `cond` alzará un error si no hay una coincidencia, para manejar esto, podemos definir una condición cuyo valor es `true`:
```

Comment:
I don't like this improvement, it sounds super awkward.
How about this one: 
```
`case` y `cond` alzarán un error si no hay una coincidencia. Para manejarlos, podemos definir una condición cuyo valor es `true`:
```

Just saying, let's debate it a little more.